fabric-dimensions-v1/src/main/java/net/fabricmc/fabric/api/dimension/v1/FabricDimensions.java

```diff
@@ -17,6 +17,7 @@
 package net.fabricmc.fabric.api.dimension.v1;
 
 import com.google.common.base.Preconditions;
+import org.jetbrains.annotations.ApiStatus;
```

Comment:
This is breaking checkstyle

Reply:
Done. I hope this can get merged soon (mainly because of that loom bump; without loom bump I cannot develop fabric or work on other prs)